src/status_im/contexts/chat/messenger/composer/effects.cljs

```diff
@@ -70,19 +70,17 @@
 
 (defn audio-effect
   [{:keys [recording? gesture-enabled?]}
-   {:keys [container-opacity]}
    audio]
   (when (and audio (not @recording?))
     (reset! recording? true)
-    (reset! gesture-enabled? false)
-    (reanimated/animate container-opacity 1)))
+    (reset! gesture-enabled? false)))
 
 (defn empty-effect
-  [{:keys [focused?]}
-   {:keys [container-opacity]}
+  [{:keys [empty-input?]}
    {:keys [input-text images link-previews? reply audio]}]
-  (when (and (not @focused?) (utils/empty-input? input-text images link-previews? reply audio))
-    (reanimated/animate-delay container-opacity constants/empty-opacity 200)))
+  (reanimated/set-shared-value
+   empty-input?
+   (utils/empty-input? input-text images link-previews? reply audio)))
```

Comment:
This was an animation, and now it is setting a shared value. Are you sure about this change @Parveshdhull ?

Reply:
hi @OmarBasem, Thank you very much for reviewing PR.

Yes, the `empty-input?` shared value is supposed to be updated whenever we have input empty without delay. We need animation for container-opacity, which is now handled in `composerContainerOpacity` in the `composer.js` file.